packages/wdio-browserstack-service/package.json

```diff
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "@wdio/logger": "^5.0.0-beta.12",
-    "browserstack-local": "^1.3.5",
+    "browserstack-local": "1.3.7",
```

Comment:
Ninja Fixing in this PR - just because i don't wanna spin up a separate one for this.